src/NuGet.Clients/VisualStudio/Extensibility/IVsFrameworkCompatibility.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains methods to discover frameworks and compatibility between frameworks.
+    /// </summary>
+    [ComImport]
+    [Guid("3B742C14-3DCB-463D-9198-F0C004BF65DD")]
+    public interface IVsFrameworkCompatibility
```

Comment:
Sorry, @joelverhagen, just getting to this now.

This interface looks good to me.